src/applications/mhv-landing-page/components/MhvRegistrationAlert.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-default
+import { default as recordEventFn } from '~/platform/monitoring/record-event';
+
+const MhvRegistrationAlert = ({ headline, recordEvent, status, icon }) => {
+  useEffect(
+    () => {
+      recordEvent({
+        event: 'nav-alert-box-load',
+        action: 'load',
+        'alert-box-headline': headline,
+        'alert-box-status': status,
+      });
+    },
+    [headline, recordEvent, status],
+  );
+
+  return (
+    <div
+      className="mhv-c-reg-alert mhv-u-reg-alert-warning usa-alert vads-u-display--flex 
+    vads-u-align-items--flex-start vads-u-justify-content--center vads-u-flex-direction--row 
+    vads-u-margin-bottom--3"
+    >
+      <va-icon icon={icon} size={4} />
+      <div className="mhv-u-reg-alert-col vads-u-flex-direction--col">
+        <h2 className="vads-u-margin-top--0 vads-u-margin-bottom--1">
+          {headline}
+        </h2>
+        <div className="mhv-u-reg-alert-body" role="presentation">
+          <p className="vads-u-margin-y--0">
+            Before you can access your messages, medications, and medical
+            records, we need to confirm your personal information and
+            relationship with VA health care. After you register, come back to
+            this page.
+          </p>
+          <p>
+            <a
+              className="vads-c-action-link--green"
+              href="https://www.myhealth.va.gov/registration"
```

Comment:
Oh, good catch, but no since the user will not have an account. What needs to happen here is that the hostname needs to be dynamic, so that needs to change.